installing/installing_ibm_z/installing-ibm-z.adoc

```diff
@@ -0,0 +1,92 @@
+[id="installing-ibm-z"]
+= Installing a cluster on IBM Z
+include::modules/common-attributes.adoc[]
+:context: installing-ibm-z
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on
+IBM Z infrastructure that you provision.
+
+[IMPORTANT]
+====
+While you might be able to follow this procedure to deploy a cluster on
+virtualized or cloud environments, you must be aware of additional
+considerations for non-bare metal platforms. Review the information in the
+link:https://access.redhat.com/articles/4207611[guidelines for deploying {product-title} on non-tested platforms]
+before you attempt to install an {product-title} cluster in such an environment.
+====
+
+.Prerequisites
+
+* Provision
```

Comment:
@dbenoit17 Can you please help with this one? Thank you!

Reply:
I may have mentioned this in my review - my apologies if I missed it.

The only supported storage for IBM Z right now is NFS.

Reply:
The architecture link is still helpful I think. While not all of it is relevant, I think it stills gives users a gist of what they should expect.

Reply:
@jaypoulz So for the first bullet point could we put: 
`Provision persistent storage using NFS for your cluster.`
Is the link still valid?

Reply:
Should we also leave there the firewall link?

Reply:
The persistent storage link is still valid since it explains Persistent Storage, so I think ensuring it says "using the NFS", so perhaps we could change the link to point to a specific section in the persistant-storage doc similar to: https://access.redhat.com/documentation/en-us/openshift_container_platform/4.2/html-single/storage/index#persistent-storage-using-nfs - instead of the overarching storage/index
The firewall link is still valid and applies to Z.

Reply:
@zzoubkova That sounds good for the first bullet. Yes the link is still valid. It's going over the basics of persistent storage, which is still relevant.

The firewall link isn't working for me in the provided PDF - can you check that it's valid? I believe it will still be relevant since routing the cluster correctly is the single most important of an OpenShift install IMHO.

Reply:
Ditto to what Andy said :)

Reply:
Fixed in commit 206ec58f58ebd974dab8235daa765b4170d740a6 

@kalexand-rh Can you please help me with checking that the links are correct? I think they are but with local build I can't see them properly. I have updated the first one to point to NFS and we need to double check the firewall one. Thank you!

Reply:
@zzoubkova, both of those links look right to me and are working in my local build.

Reply:
@kalexand-rh perfect!